internal/databases/redis/backup_push_handler.go

```diff
@@ -0,0 +1,80 @@
+package redis
+
+import (
+	"bufio"
+	"github.com/go-redis/redis"
+	"github.com/wal-g/wal-g/internal"
+	"github.com/wal-g/wal-g/internal/tracelog"
+	"github.com/wal-g/wal-g/utility"
+	"math"
+	"os"
+	"time"
+)
+
+func HandleBackupPush(uploader *Uploader) {
+	// Configure folders
+	redisDataFoler, ok := internal.GetSetting("WALG_REDIS_DATA_FOLDER")
+	if !ok {
+		redisDataFoler = "/var/lib/redis"
+	}
+	uploader.UploadingFolder = uploader.UploadingFolder.GetSubFolder(utility.BaseBackupPath)
+
+	// Get Redis connection
+	client := getRedisConnection()
+
+	// Init backup process
+	currentTime := time.Now()
+	initializeBackupSaving(client)
+
+	// Wait for new backup on disk.
+	// DISCUSS: Maybe add Timer to stop trying after some time?
```

Comment:
1. We discussed Redis dumping to stdout, is there something wrong with stream-push method?
2. Can't we await for this event using redis library?
3. If everything abovementioned doesn't work, then let's do it.